example/lib/payments/widget/payments_page.dart

```diff
@@ -0,0 +1,178 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:kevin_flutter_example/common_widgets/kevin_button.dart';
+import 'package:kevin_flutter_example/common_widgets/kevin_check_box.dart';
+import 'package:kevin_flutter_example/common_widgets/kevin_form_field.dart';
+import 'package:kevin_flutter_example/common_widgets/kevin_progress_indicator.dart';
+import 'package:kevin_flutter_example/common_widgets/kevin_snack_bar.dart';
+import 'package:kevin_flutter_example/payments/bloc/payments_bloc.dart';
+import 'package:kevin_flutter_example/payments/model/country_item.dart';
+import 'package:kevin_flutter_example/payments/model/creditor_list_item.dart';
+import 'package:kevin_flutter_example/theme/app_images.dart';
+import 'package:kevin_flutter_example/theme/widget/app_theme.dart';
+import 'package:kevin_flutter_example/web/app_urls.dart';
+import 'package:kevin_flutter_example/web/external_url.dart';
+import 'package:scroll_snap_list/scroll_snap_list.dart';
+
+part 'payments_body_widgets.dart';
+
```

Comment:
Maybe remove new lines between `part`s (same as import)?

Reply:
It's an auto formatting, so will leave it like that to avoid unnecessary line changes in the future.